<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edithCpDialog</class>
 <widget class="QDialog" name="edithCpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="EdithGenius">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Edit Genius&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGeniusName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGeniusNameError">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGeniusBirthYearError">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Birth Year: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGeniusDeathYearError">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Death Year:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBirthYear">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDeathYear">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGeniusGender">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>351</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGeniusNameError_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenderError">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGeniusBirthYearError_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGeniusDeathYearError_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
